### PR TITLE
Fix PHP 8 warning

### PR DESCRIPTION
**Pull request by zonky2**
Fix PHP 8 warning

### CODE REVIEW

File: core-bundle/src/Resources/contao/dca/tl_content.php [hunk 1]
```diff
@@ -1736,7 +1736,7 @@ public function getArticles(DataContainer $dc)
 			while ($objArticle->next())
 			{
 				$key = $objArticle->parent . ' (ID ' . $objArticle->pid . ')';
-				$arrArticle[$key][$objArticle->id] = $objArticle->title . ' (' . ($GLOBALS['TL_LANG']['COLS'][$objArticle->inColumn] ?: $objArticle->inColumn) . ', ID ' . $objArticle->id . ')';
```
Comment (leofeyer) [inline core-bundle/src/Resources/contao/dca/tl_content.php:1739]: The same code is also in line 1416 and in `tl_news.php:811`. Why did you only fix it here? Plus there are two occurrences of `] ?: $` in `DC_Table` and `DC_Folder` that we should check as well.

Comment (zonky2) [reply core-bundle/src/Resources/contao/dca/tl_content.php:1739]: I only got a warning at the point

Comment (leofeyer) [reply core-bundle/src/Resources/contao/dca/tl_content.php:1739]: I cannot merge the PR unless all occurrences of the problem have been fixed. 🤷‍♂️

**Comment by leofeyer** [comment]
Closed in favor of #6161.

**Pull request closed without merging**